docs/stylesheets/theme.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --md-primary-fg-color:        #FA915A;
+  --md-accent-fg-color:         #DB5C20;
+  --md-primary-fg-color--light: #ECB7B7;
+  --md-primary-fg-color--dark:  #FF7E54;
+}
+
+@media (prefers-color-scheme: dark) {
```

Comment:
This adds a custom dark mode support. When the browser tells that he is in dark-mode, the website will be displayed dark as well. [This website](https://facelessuser.github.io/pymdown-extensions/) uses the same tech-stack (mkdocs, material theme) and has an even cooler dark-mode, including a switch to turn it off. But I've not figured out how it works, yet.

Reply:
we will, sooner or later, or get help that know how to do that. 
Important is that we know now, thanks to you, that this feature exits!